scripts/bert/embedding.py

```diff
@@ -103,22 +103,20 @@ def embedding(self, sentences, oov_way='avg'):
 
         Returns
         -------
-        List[(ndarray, List[str], List[ndarray])]
-            List of sentence embedding, tokens, and tokens embedding
+        List[(List[str], List[ndarray])]
+            List of tokens, and tokens embedding
         """
         data_iter = self.data_loader(sentences=sentences)
         batches = []
         for token_ids, valid_length, token_types in data_iter:
             token_ids = token_ids.as_in_context(self.ctx)
             valid_length = valid_length.as_in_context(self.ctx)
             token_types = token_types.as_in_context(self.ctx)
-            sequence_outputs, pooled_outputs = self.bert(token_ids,
-                                                         token_types,
-                                                         valid_length.astype('float32'))
-            for token_id, sequence_output, pooled_output in zip(token_ids.asnumpy(),
-                                                                sequence_outputs.asnumpy(),
-                                                                pooled_outputs.asnumpy()):
-                batches.append((token_id, sequence_output, pooled_output))
+            sequence_outputs = self.bert(token_ids, token_types,
+                                         valid_length.astype('float32'))
+            for token_id, sequence_output in zip(token_ids.asnumpy(),
+                                                 sequence_outputs.asnumpy()):
+                batches.append((token_id, sequence_output))
```

Comment:
Yes, `pooled_outputs` is a tensor of pooled representation of the first tokens. But we have already got `sequence_outputs ` here, can we use some methods to fuse `sequence_outputs` to get the basic sentence representation?

Reply:
My navie idea is to average all the token embeddings. The performance should be tested, though. 
@szha or should I create another issue for this. This fix pull request can be merged first.

Reply:
I'd lean towards returning sequence outputs and the let the consumer decide how to pool it. This way, the separation of concern is clear.